### PR TITLE
data: SahilSinghTomar

### PR DESCRIPTION
**Pull request by SahilSinghTomar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6089"></a>

### CODE REVIEW

File: data/SahilSinghTomar.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": Sahil Singh Tomar,
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/SahilSinghTomar.json:3]: this is no longer required
```suggestion
```

File: data/SahilSinghTomar.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "type": "personal",
+  "bio": "Learning Full Stack development",
+  "avatar": "https://github.com/SahilSinghTomar.png",
```
Comment (SaraJaoude) [inline data/SahilSinghTomar.json:6]: this is no longer required
```suggestion
```

File: data/SahilSinghTomar.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": Sahil Singh Tomar,
```
Comment (kumarsonsoff3) [inline data/SahilSinghTomar.json:2]: Also, wrap your name in inverted commas as `"Sahil Singh Tomar"`.

```suggestion
  "name": "Sahil Singh Tomar",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! :tada: 

--
In case you might want to add more links or update your profile, you can follow these [editing docs](https://linkfree.io/docs/how-to-guides/editing).

**Pull request merged**